Source/WebKit/Shared/Cocoa/WKNSArray.mm

```diff
@@ -52,7 +52,7 @@ - (NSUInteger)count
 - (id)objectAtIndex:(NSUInteger)i
 {
     API::Object* object = _array->at(i);
-    return object ? object->wrapper() : [NSNull null];
+    return object ? (id)object->wrapper() : [NSNull null];
```

Comment:
I am surprised that the cast to `id` is needed. What kind of error do you get without it?

Reply:
Doing `(id)[NSNull null]` works too.

Reply:
This needs to use a proper `(__bridge id)` cast to work with ARC later.  Please use `bridge_id_cast()` from `<wtf/cocoa/TypeCastsCocoa.h>`:

```
    return object ? bridge_id_cast(object->wrapper()) : [NSNull null];
```

I think casting `object->wrapper()` is preferred here since it's required once this file is built with ARC.

Reply:
`wrapper()` is already doing a `(__bridge id <WKObject>` cast in it. This is just a secondary cast. Does that matter?

Reply:
Oops, `wrapper()` already returns `id <WKObject>`.  Got confused scrolling up and down patch.

So this doesn't need to be changed to enable ARC later.  Using `(id)` cast is fine.